docs/docs/configuration/networking-settings.mdx

```diff
@@ -27,6 +28,59 @@ Note that Superset bundles [flask-talisman](https://pypi.org/project/talisman/)
 Self-descried as a small Flask extension that handles setting HTTP headers that can help
 protect against a few common web application security issues.
 
+
+## HTML Embedding of Dashboards and Charts
+
+There are two ways to embed a dashboard: Using the [SDK](https://www.npmjs.com/package/@superset-ui/embedded-sdk) or embedding a direct link. Note that in the latter case everybody who knows the link is able to access the dashboard.
+
+### Enabling Embedding via the SDK
+
+Clicking on `...` next to `EDIT DASHBOARD` on the top right of the dashboard's overview page should yield a drop-down menu including the entry "Embed dashboard".
```

Comment:
If I follow correctly, I don't see this option on my Superset because I haven't yet enabled this Superset config feature.  If that's right, could this instruction be moved below the feature flag instruction, so that users don't try this before it's possible?